frameworks/Java/javalin/src/main/java/com/lostsys/test/javalin/Bench.java

```diff
@@ -0,0 +1,30 @@
+package com.lostsys.test.javalin;
+
+import io.javalin.Javalin;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class Bench {
+
+    public static void main(String[] args) {
+
+        Javalin app = Javalin.create().start(8080);
+        app.get("/plaintext", ctx -> ctx.result("Hello, World!"));
+
+        app.get("/json", ctx -> {
+            Map<String, String> hello = new HashMap<>();
+            hello.put("message", "Hello, World!");
+            ctx.json(hello);
+        });
+
+    }
+
+    private static Map<String, String> sayHello() {
```

Comment:
Remove this or use it.  IMO it's simple enough to be inlined; no need for a helper method.